tracer/src/Datadog.Trace/DataStreamsMonitoring/Hashes/HashHelper.cs

```diff
@@ -32,6 +32,11 @@ public static NodeHashBase CalculateNodeHashBase(string service, string? env, st
             hash = FnvHash64.GenerateHash(primaryTag!, HashVersion, hash);
         }
 
+        if (!string.IsNullOrEmpty(processTags))
+        {
+            hash = FnvHash64.GenerateHash(processTags!, HashVersion, hash);
```

Comment:
nit: if you use `StringUtil.IsNullOrEmpty` you don't need the null-forgiving to keep .NET Framework happy 🙂 (I think you need to add an extra `using` though)
```suggestion
        if (!StringUtil.IsNullOrEmpty(processTags))
        {
            hash = FnvHash64.GenerateHash(processTags, HashVersion, hash);
```

Reply:
ah good to know thanks, I'll do that